test/unit/org/apache/cassandra/repair/FuzzTestBase.java

```diff
@@ -534,12 +537,6 @@ private static void findCorrectRange(MerkleTrees trees, Token token, Consumer<Me
         }
     }
 
-    private enum RepairType
```

Comment:
to make the repair args generation reused rather than copy pasted, took this logic and moved it to `RepairGenerators`